src/api/app/controllers/webui/package_controller.rb

```diff
@@ -698,21 +698,26 @@ def save_file
       # We have to check if it's an AJAX request or not
       if request.xhr?
         flash.now[:success] = message
-        render layout: false, partial: 'layouts/webui/flash', object: flash
+        status = 200
```

Comment:
why is this needed? is not it the default? :confused:

Reply:
I do this to avoid having two times a `render` call. I do it instead later with this `status` variable.

Reply:
yes, but you don't need to do this variable. As it is the default you could do something like:

```ruby
params = { layout: false, status: status, partial: "layouts/#{namespace}/flash", object: flash }
params[:status] = status if defined? status
render params
```

or the cool version:

```ruby
render layout: false, status: (status ||= 200), partial: "layouts/#{namespace}/flash", object: flash
```

Reply:
I like the second version